src/mappings/util.ts

```diff
@@ -165,21 +173,27 @@ export const getSigner = (extrinsic: SubstrateExtrinsic): string => {
 export const getDocValue = (
   doc: Codec
 ): Pick<AssetDocument, 'name' | 'link' | 'contentHash' | 'type' | 'filedAt'> => {
-  const {
-    uri: link,
-    content_hash: documentHash,
-    name,
-    doc_type: type,
-    filing_date: filedAt,
-  } = JSON.parse(doc.toString());
+  const document = JSON.parse(doc.toString());
+
+  const documentHash = extractValue(document, 'content_hash');
 
   const hashType = Object.keys(documentHash)[0];
+  const contentHash = {
+    type: hashType,
+    value: maybeHexToString(documentHash[hashType]),
```

Comment:
I don't think `maybeHexToString` is safe for hash contents. I think they should be left as hex since I think thats what they are normally represented as. (ASCII byte shouldn't start with `1`, UTF-8 has more complicated rules.)

Not all `byte[]` are valid strings, and I think this will produce unexpected results at times.

e.g. hex from `shasum`
```js
const s = '548cec21939ade4e88624d3b44959a8a3fd2b076'
 for (let i = 0; i < s.length; i += 2)
    str += String.fromCharCode(parseInt(s.substr(i, 2), 16));
'\x00\x00\x00\x00\x00\x00T\x8Cì!\x93\x9AÞN\x88bM;D\x95\x9A\x8A?Ò°v'
```